active/ZEP0000.md

```diff
@@ -60,29 +64,53 @@ that require collaboration across multiple projects.
-should be adopted by every implementation of Zarr and, in general, the overall
-Zarr community. Core ZEPs must go through a thorough review process, including
-involvement, discussion, and voting from the author of the proposed ZEP, Zarr
-Steering Council, author(s) of various Zarr implementations (ZIC), and
-open-source projects/research groups using Zarr and the general Zarr community.
-The general advice is that everyone should be made aware of changes introduced
-by core ZEPs. Core protocol ZEPs require community consensus, and developers or
-users are typically not free to ignore them.
-
-Note: Currently, Specification ZEPs only deal with the core specification of Zarr.
-The scope of SPEC ZEPs will be extended in the future.
+[zarr.dev/implementations]. The core protocol ZEPs should be adopted by every
+implementation of Zarr and, in general, the overall Zarr community. Core ZEPs
+must go through a thorough review process, including involvement, discussion,
+and voting from the author of the proposed ZEP, Zarr Steering Council, author
+(s) of various Zarr implementations (ZIC), and open-source projects/research
+groups using Zarr and the general Zarr community. The general advice is that
+everyone should be made aware of changes introduced by core ZEPs. Core protocol
+ZEPs require community consensus, and developers or users are typically not
+free to ignore them.
+
+- **Protocol extension ZEP**
+
+Describes a ZEP which involves addition to the extension of the core
+specification protocol for Zarr. Protocol extension ZEPs (commonly known as
+extension ZEPs) are part of Spec ZEPs and apply to Zarr storage extension
+specification and various implementations under the Zarr Implementations. The
+extension ZEPs are designed so that they are not necessary to be adopted by
+every other Zarr implementation. The extension ZEPs may or may not include
+involvement from the author of various Zarr implementations and the general
+Zarr community. The general idea of having an extension ZEP is to add
+functionality, data types and features to accommodate emerging and essential
+use cases.
+
+- **Lean ZEP**
+
+Describes a ZEP proposing a significantly small and straightforward change to
+the Zarr specification. Lean ZEPs are designed for minor modifications that do
+not require an extensive review process or community consensus. These proposals
+aim to streamline the introduction of small improvements or corrections to the
+Zarr specification. Unlike other ZEP types, Lean ZEPs may be introduced through
+a simplified process, typically by submitting a GitHub pull request directly.
+Lean ZEPs focus on non-controversial changes with minimal impact on existing
+implementations, encouraging community input while prioritizing swift
+implementation and agility. In some cases, Lean ZEPs may include breaking
+changes if there is unanimous approval from all participating stakeholders,
+including the ZIC and the broader Zarr community.
```

Comment:
This section is unclear on what a lean ZEP actually is (in terms of procedure), and what contributors have to in the lean ZEP process. 

- Is a lean ZEP literally any small PR against the zarr specs repo? In this case, what is the utility of describing this as a ZEP? This reasoning should be stated explicitly, because it is not obvious. 
- Is a lean ZEP _more_ than a small PR against the zarr specs repo, e.g. a PR + some summary document on a web page? If so, who writes that summary document? Is it extra work for the person submitting the PR? This should also be stated explicitly in this text.

Reply:
closing this, because the content I'm looking for is later on in the document :)